components/Player/CoverArt/CoverArt.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file CoverArt.tsx
+ * Cover art component for player.
+ */
+
+import { useContext } from 'react';
+import PrxImage from '@components/PrxImage';
+import PlayerContext from '@contexts/PlayerContext';
+import styles from './CoverArt.module.scss';
+
+const CoverArt = () => {
+  const { state } = useContext(PlayerContext);
+  const { tracks, currentTrackIndex } = state;
+  const { imageUrl, title } = tracks[currentTrackIndex];
+
+  return (
+    imageUrl && (
+      <div className={styles.coverArt}>
+        <PrxImage
+          src={imageUrl}
+          alt={`Cover art for "${title}".`}
```

Comment:
It doesn't look like the alt text is working.

Reply:
For what its worth, I like this approach :)

Reply:
Good catch. We just forgot to add this attribute to the Next.js image component. Just pushed up a fix. Alt text should be in the img tag now.